docs/modules/ROOT/pages/how-to-guides/testing_applications/retriggering_integration_tests.adoc

```diff
@@ -0,0 +1,13 @@
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+== Prerequisites
+The snapshot for which tests will be retriggered has completed its initial tests
+
+== Procedure
+. Determine which IntegrationTestScenario should be re-run.  For a given snapshot, only one scenario can
+be re-run at a time.
+. Add the label `test.appstudio.openshift.io/run` to the snapshot. Set the value to the name of the scenario to be re-run
+. The tests will be re-triggered automatically.  Once they are re-triggered the label will be removed.  At this point you
```

Comment:
Here maybe say "use `kubectl get pipelineruns`" or `tkn pr list` to confirm that your test pipelinerun was successfully re-triggered"?